doc/subsystems/bluetooth/qualification.rst

```diff
@@ -15,3 +15,28 @@ the following documents:
    l2cap-pics.rst
    sm-pics.rst
    rfcomm-pics.rst
+
+Qualification Listings
+**********************
+
+The Zephyr BLE stack has obtained qualification listings for both the Host
+and the Controller.
+See the tables below for a list of qualification listings
+
+Host qualifications
+-------------------
+
+=============== ======================================================= ===================
+Zephyr version  Link                                                    Qualifying Company
+=============== ======================================================= ===================
+1.13            https://launchstudio.bluetooth.com/ListingDetails/70189 Nordic
```

Comment:
These links to launchstudio require a login/account to see any details :(

Reply:
They do? that is strange. OK I will figure it out

Reply:
I can see the page you're linking to, but if I click on the details link at the bottom of the page, it requires me to login:

![image](https://user-images.githubusercontent.com/11063618/47925657-f8202500-de7b-11e8-87a6-159cf8fe2ea0.png)

Reply:
why is this a concern? It is not on a Zephyr page. The landing page is the important one.